packages/styles/src/components/TearSheet/tear-sheet-wrapper.scss

```diff
@@ -1,39 +1,53 @@
 .#{$iot-prefix}--tear-sheet-wrapper {
   top: 0;
   left: 0;
-  height: 100%;
+  height: 100vh;
```

Comment:
just checking, in security's version of tearsheet I see this

![image](https://user-images.githubusercontent.com/14989804/123117649-efefcc00-d40f-11eb-986b-18f36450adfa.png)

just a note

Reply:
https://ibm-security.carbondesignsystem.com/?path=/story/patterns-tearsheet--basic

Reply:
Interesting 🤔 @davidicus this is probably something to check out as part of a new issue?